manifests/interface.pp

```diff
@@ -134,6 +132,18 @@
     'virtual'            => $virtual,           # undef, bool
   }
 
+  $legacy_bind_variables = {
+    'any-ipv4-address'   => $any_ipv4_address,  # undef, bool
+    'any-ipv6-address'   => $any_ipv6_address,  # undef, bool
+  }
+
+  if $jboss::product == 'wildfly' and versioncmp($jboss::version, '9.0.0') >= 0 {
+    $bind_variables = $basic_bind_variables
```

Comment:
We should throw here a deprecated warning if values are set to something other than undef and using newer versions of products